azure-functions/acbs-function/mappings/party/exporter.js

```diff
@@ -11,24 +15,38 @@ Field mapping based on email from Gareth Ashby 15/03/2021
   sme                       string  Workflow sme type code  4 digit code - default if unknown 70
   citizenshipClass          string  If Customer domicile country is UK set to '1' otherwise '2'
   officerRiskDate           date    yyyy-MM-dd i.e. 2019-10-21, Date of creation (we use current date)
-*/
+  */
 
 const exporter = ({ deal, acbsReference }) => {
-  const { submissionDetails } = deal.dealSnapshot;
+  // Get Deal's Snapshot
+  const submissionDetails = deal.dealSnapshot;
+
+  // Get Product Type i.e. GEF
+  const product = deal.dealSnapshot.dealType;
 
-  const countryCode = submissionDetails['supplier-address-country'] && submissionDetails['supplier-address-country'].code;
-  const citizenshipClass = countryCode === 'GBR' ? '1' : '2';
+  if (product === CONSTANTS.PRODUCT.TYPE.GEF) {
+    countryCode = submissionDetails.exporter.registeredAddress.country;
+    dealCountry = 'GB';
+  } else {
+    countryCode = submissionDetails['supplier-address-country'] && submissionDetails['supplier-address-country'].code;
+    dealCountry = 'GBR';
+  }
 
-  const partyNames = getPartyNames(submissionDetails['supplier-name']);
+  const citizenshipClass = countryCode === dealCountry ? '1' : '2';
```

Comment:
could we put 1 and 2 values into a constants? I'm not sure what it means, maybe something like this?
```
CONSTANTS.ACBS.CITIZENSHIP_CLASS.SAME_AS_DEAL.COUNTRY
CONSTANTS.ACBS.CITIZENSHIP_CLASS.DEFAULT

```